modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -127,7 +127,9 @@ export default class LargeVideoManager {
             // the video was not rendered, before the connection has failed.
             const isHavingConnectivityIssues
                 = APP.conference.isParticipantConnectionActive(id) === false;
-            if (isHavingConnectivityIssues
+            const isInLastN
+                = APP.conference.isLocalId(id) || APP.conference.isInLastN(id);
+            if ( (isHavingConnectivityIssues || !isInLastN)
```

Comment:
Isn't this for the Large Video? How somebody can be displayed on the large video and not be part of last n?